locale/uk/security.md

```diff
@@ -33,16 +40,14 @@ title: Безпека
 
 - Цей процес може зайняти деякий час, особливо коли необхідна координація з мейнтейнерами інших проектів. Всі зусилля будуть спрямовані на те, щоб вирішити проблему настільки швидко, наскільки це можливо, однак, важливо, щоб ми дотримувались процесу релізу, щоб бути певними, що все відбувається належним чином.
 
-
 ## Отримання оновлень безпеки
 
 Сповіщення, що стосуються безпеки, будуть поширюватись за допомогою:
 
-- [https://groups.google.com/group/nodejs-sec](https://groups.google.com/group/nodejs-sec)
-- [https://nodejs.org/en/blog](https://nodejs.org/en/blog)
-
+- <https://groups.google.com/group/nodejs-sec>
```

Comment:
will this produce correct links in html?

Reply:
I've copied from the English source.

Reply:
As per the [the English source](https://nodejs.org/en/security/), it seems so.